tests/test_version.py

```diff
@@ -33,10 +33,22 @@ def test_parse(version, klass):
     assert isinstance(parse(version), klass)
 
 
+class VersionObj(object):
+    def __init__(self, version):
+        self.version = version
+
+    def __str__(self):
+        return self.version
+
+    def __repr__(self):
+        return repr(self.version)
+
```

Comment:
FWIW, this may be a bug in pep8 the tool. Guido has filed a couple bugs around this.